lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -108,10 +116,8 @@ export default async function({workspace, taskUtil, options = {}}) {
 			log.verbose("Writing flexibility changes bundle");
 			return workspace.write(resource);
 		}));
-		// Add the sap.ui.fl dependency if a bundle has been created
-		if (processedResources.length > 0) {
-			await updateFLdependency();
-		}
+		// Add the sap.ui.fl dependency if a bundle has been created and flag if a bundle was created
```

Comment:
I feel like this comment doesn't relate to the code anymore. At least the added `and flag if a bundle was created` is not related to it.

Reply:
Thanks!
It's actually partially valid- it will add the `sap.ui.fl` dependency in case the value is `true`.
I have updated it